scripts/goldpolish-target-extract-seq.py

```diff
@@ -6,13 +6,22 @@
 import btllib
 
 
+class Coordinate:
```

Comment:
Do you ever need to mutate these? If not, you should instead make this a namedtuple